chrome/content/zotero/preferences/preferences_export.xul

```diff
@@ -49,10 +50,15 @@
 			<label value="&zotero.preferences.quickCopy.defaultOutputFormat;" control="quickCopy-menu"/>
 			<menulist id="zotero-quickCopy-menu"/>
 			
-			<separator class="thin"/>
-			
-			<checkbox id="zotero-quickCopy-copyAsHTML" label="&zotero.preferences.quickCopy.copyAsHTML;"
-				oncommand="Zotero_Preferences.Export.buildQuickCopyFormatDropDown(document.getElementById('zotero-quickCopy-menu'), this.checked ? 'html' : '');"/>
+			<hbox align="center">
+				<label value="&zotero.preferences.quickCopy.locale;" control="locale-menu"/>
+				<menulist id="locale-menu"/>
```

Comment:
Other windows are mostly fine (I think). Just the preference window should
follow the convention. I don't see the problem with setting this to what I
mentioned in my first comment. The code that deals with this section of the
preference pane doesn't deal with other locale menus.

On Wed, May 6, 2015 at 9:00 PM, Rintze M. Zelle <notifications@github.com>
wrote:

> In chrome/content/zotero/preferences/preferences_export.xul
> <https://github.com/zotero/zotero/pull/623#discussion_r29819501>:
>
> > @@ -49,10 +50,15 @@
> >  			<label value="&zotero.preferences.quickCopy.defaultOutputFormat;" control="quickCopy-menu"/>
> >  			<menulist id="zotero-quickCopy-menu"/>
> >  			
> > -			<separator class="thin"/>
> > -			
> > -			<checkbox id="zotero-quickCopy-copyAsHTML" label="&zotero.preferences.quickCopy.copyAsHTML;"
> > -				oncommand="Zotero_Preferences.Export.buildQuickCopyFormatDropDown(document.getElementById('zotero-quickCopy-menu'), this.checked ? 'html' : '');"/>
> > +			<hbox align="center">
> > +				<label value="&zotero.preferences.quickCopy.locale;" control="locale-menu"/>
> > +				<menulist id="locale-menu"/>
>
> Would "zotero-locale-menu" work? Then at least we're in the "zotero"
> namespace. I don't see an easy fix to give the locale menulists of various
> windows their own unique ids.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zotero/zotero/pull/623/files#r29819501>.
>